src/traces/icicle/attributes.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+var hovertemplateAttrs = require('../../plots/template_attributes').hovertemplateAttrs;
+var texttemplateAttrs = require('../../plots/template_attributes').texttemplateAttrs;
+
+var colorScaleAttrs = require('../../components/colorscale/attributes');
+var domainAttrs = require('../../plots/domain').attributes;
+var pieAttrs = require('../pie/attributes');
+var sunburstAttrs = require('../sunburst/attributes');
+var treemapAttrs = require('../treemap/attributes');
+var constants = require('../treemap/constants');
+var extendFlat = require('../../lib/extend').extendFlat;
+
+module.exports = {
+    labels: sunburstAttrs.labels,
+    parents: sunburstAttrs.parents,
+
+    values: sunburstAttrs.values,
+    branchvalues: sunburstAttrs.branchvalues,
+    count: sunburstAttrs.count,
+
+    level: sunburstAttrs.level,
+    maxdepth: sunburstAttrs.maxdepth,
+
+    tiling: {
+        orientation: {
+            valType: 'enumerated',
+            values: ['v', 'h'],
+            dflt: 'h',
+            editType: 'plot',
+            description: [
+                'Sets the orientation of the icicle.',
+                'With *v* the icicle grows vertically.',
+                'With *h* the icicle grows horizontally.',
+            ].join(' ')
+        },
+
+        flip: treemapAttrs.tiling.flip,
+
+        pad: treemapAttrs.tiling.pad,
```

Comment:
I think the default value for this attribute should be 0, so as to look more like sunbursts and less like treemaps

Reply:
Yes that's a good idea as well. I remember when playing around with padding, 0 or close-to-0 looked the best to me.

Will change this.